### PR TITLE
Replace mutable var in ChpModelSpec

### PR DESCRIPTION
**Pull request by pierrepetersmeier**
Replace mutable var in ChpModelSpec

Resolves: #1002

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -91,6 +91,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Rewrote SystemParticipantTest and MockParticipant from groovy to scala [#646](https://github.com/ie3-institute/simona/issues/646)
 - Rewrote ChpModelTest from groovy to scala [#646](https://github.com/ie3-institute/simona/issues/646)
 - Rewrote CylindricalThermalStorageTest Test from groovy to scala [#646](https://github.com/ie3-institute/simona/issues/646)
+- Replace mutable var in ChpModelSpec [#1002](https://github.com/ie3-institute/simona/issues/1002)
+- Rewrote CylindricalThermalStorageTest Test from groovy to scala [#646](https://github.com/ie3-institute/simona/issues/646)
```
Comment (danielfeismann) [inline CHANGELOG.md:95]: Please be carefule when merging the dev. This entry doubled, see line 93.

File: src/test/scala/edu/ie3/simona/model/participant/ChpModelSpec.scala [hunk 2]
```diff
@@ -84,7 +80,7 @@ class ChpModelSpec
       getQuantity(0, KILOWATT),
     )
 
-    chpInput = new ChpInput(
+    val chpInput = new ChpInput(
       UUID.randomUUID(),
       "ChpInput",
       null,
```
Comment (danielfeismann) [inline src/test/scala/edu/ie3/simona/model/participant/ChpModelSpec.scala:86]:
```suggestion
      OperatorInput.NO_OPERATOR_ASSIGNED,
```

File: CHANGELOG.md (hunk 1 above)
Comment (pierrepetersmeier) [reply CHANGELOG.md:95]: Oh Yes, Thanks 👍🏻

**Review by danielfeismann — APPROVED** [review]
LGTM

**Pull request merged**